### ISSUE TITLE
AUR package

### ISSUE THREAD

**Issue by Bugreporterbug** [sdaqo/anipy-cli#188]
**Describe the the new feature**
In getting started there's mention of AUR packages that coming soon. 
How soon is soon?

**Comment by sdaqo** [comment]
Yeah you are right...  I just looked into it, but I probably will not be making one. I would have to maintain and create PKGBUILDs for several independent dependencies that do not have aur/arch packages and honestly, I would rather not. 
I am currently not using arch and honestly do not see the value in an aur package, because we already have a pretty good ways to install it through pipx or nix.

If anyone wants to make one I wont stop them, but I will take it out of the docs for now, sorry.
